src/etc/inc/interfaces.inc

```diff
@@ -2987,11 +2990,23 @@ function interface_dhcpv6_prepare($interface = 'wan', $wancfg)
         } else {
             log_error("DHCP6 config file override does not exist: '{$dhcp6cfile}'");
         }
+    } else {
```

Comment:
if you pull from this branch my changes should land in your git.

As for the else here we have to consider the "override", move it to the proper position where we would save our file and then merge it into the others (no exception).

dhcp6c_set_scripts() isn't really needed so far as we do not change the call flow. Nicer to have it in one function like before.

Reply:
Problem there is that if I pull then it overwrites my stuff. So now I am doing my bits in a separate file and merging them after.

Reply:
ok, sorry for the trouble. not sure what is quicker for collaboration, but we are almost ready for merge by the looks of the result :)